Test/git-issues/git-issue-3333.dfy.expect

```diff
@@ -0,0 +1,2 @@
+git-issue-3333.dfy(8,18): Error: Function body type mismatch (expected uint16, got int)
```

Comment:
At some point it will be good to actually say in this case "To declare a constant integer as a uint16, write "1 as uint16x"", but I guess it's out of this scope.